utilities/copy_fastq.jst

```diff
@@ -12,7 +12,7 @@
 
     mkdir -p temp/fastqs/
 
-    {% if fastq.petagene == true %}
+    {% if fastq.fileType == "fasterq" %}
     rsync "{{ fastq.fastqPath|replace(".fastq.gz",".fasterq") }}" "temp/fastqs/"
```

Comment:
@bryce-turner why is the replace needed, in the YAML would the fastq.fastqPath not have the .fasterq extension anyways?

Reply:
I'll retest this, but I think the working idea at the time was that the .fastq.gz would not be visible, or wouldn't copy everything correctly. 

Along these lines I think we were attempting to minimize changes to fastq pathing logic in LIMS. E.g. the only control necessary in LIMS for tracking fasterq was to use the fileType, not the pathing.